node/http2.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+
+import { notImplemented } from "./_utils.ts";
+
+export class Http2Session {
+  constructor() {
+    notImplemented();
+  }
+}
+export class ServerHttp2Session {
+  constructor() {
+    notImplemented();
+  }
+}
+export class ClientHttp2Session {
+  constructor() {
+    notImplemented();
+  }
+}
+export class Http2Stream {
+  constructor() {
+    notImplemented();
+  }
+}
+export class ClientHttp2Stream {
+  constructor() {
+    notImplemented();
+  }
+}
+export class ServerHttp2Stream {
+  constructor() {
+    notImplemented();
+  }
+}
+export class Http2Server {
+  constructor() {
+    notImplemented();
+  }
+}
+export class Http2SecureServer {
+  constructor() {
+    notImplemented();
+  }
+}
+export function createServer() {}
+export function createSecureServer() {}
+export function connect() {}
+export const constants = undefined;
```

Comment:
```suggestion
export const constants = {};
```
(just a suggestion, `undefined` isn't really wrong either)